xbmc/interfaces/json-rpc/VideoLibrary.cpp

```diff
@@ -77,7 +77,7 @@ JSONRPC_STATUS CVideoLibrary::GetMovies(const std::string &method, ITransportLay
     setID = 0;
 
   CFileItemList items;
-  if (!videodatabase.GetMoviesNav(videoUrl.ToString(), items, genreID, year, -1, -1, -1, -1, setID, -1, sorting, RequiresAdditionalDetails(MediaTypeMovie, parameterObject)))
+  if (!videodatabase.GetMoviesNav(videoUrl.ToString(), items, genreID, year, -1, -1, -1, -1, setID, -1, -1, sorting, RequiresAdditionalDetails(MediaTypeMovie, parameterObject)))
```

Comment:
Side note: Not invented by you, so nothing to change, but this is the most horrible interface I have ever seen. So many parameters, magic numbers saying nothing to the reader everywhere. Just unmaintainable.